src/main/webapp/script.js

```diff
@@ -36,6 +39,9 @@ function createTable() { // eslint-disable-line no-unused-vars
     }
   });
 }
+
+const checkMap = new Map();
```

Comment:
You never use the keys when accessing elements of `checkMap`. Should it just be an `Array`?